extension/googleclientauthextension/config.go

```diff
@@ -36,13 +57,24 @@ type Config struct {
 	// See https://datatracker.ietf.org/doc/html/rfc6749#section-3.3
 	Scopes []string `mapstructure:"scopes,omitempty"`
 
+	// TokenFormat specifies which type of token will be generated.
+	// default: access_token
+	TokenFormat string `mapstructure:"token_format,omitempty"`
```

Comment:
Nit: should this be token_type? https://cloud.google.com/docs/authentication/token-types

Reply:
renamed. :+1: